spec/index.html

```diff
@@ -299,6 +299,74 @@ <h3>Object Lists</h3>
     Turtle provides a number of ways of writing each.
   </p>
 
+  <section id="sec-version">
+    <h3>Version Announcement</h3>
+
+    <p>The Turtle language has evolved since its origin, and RDF 1.2
+      introduces new features which may confuse non-compliant parsers.
+      RDF 1.2 Turtle introduces the
+      <a href="#grammar-production-sparqlVersion"><code>VERSION</code></a> and
+      <a href="#grammar-production-version"><code>@version</code></a> directives
+      along with an optional `version` <a href="#sec-mediaReg">Media Type parameter</a>.
+      When serializing Turtle with new features such as base direction or triple terms,
+      authors can announce the presense of these features using these directives.
+    </p>
+
+    <pre id="ex-version-decl"
+         class="example turtle" data-transform="updateExample"
+         title="Version announcement">
+      <!--
+      @version "1.2" .
+      @prefix : <http://example.com/> .
+      :a :name "Alice" ~ :t {| :statedBy :bob ; :recorded "2021-07-07"^^xsd:date |} .
+      -->
+    </pre>
+
+    <p>Alternatively, the SPARQL style directive can be used
+      (note that there is no trailing <a href="#cp-full-stop"><code title="full stop">.</code></a>):</p>
+
+    <pre id="ex-sparql-version-decl"
+         class="example turtle" data-transform="updateExample"
+         title="SPARQL-style version announcement">
+      <!--
+      VERSION "1.2"
+      PREFIX : <http://example.com/>
```

Comment:
```suggestion
      @version "1.2" .
      @prefix : <http://example.com/> .
```

Reply:
If you're suggesting that the SPARQL-style `VERSION` example should show before the old-style `@version` example, then this is at odds with the example ordering in the IRI section.

Reply:
It's in the commits.